el/index.js

```diff
@@ -38,6 +38,13 @@ module.exports = yeoman.generators.Base.extend({
         'ex: yo polymer:el my-element'
       ));
     }
+    
+    var isInsideProjectFolder = this.fs.exists('app/elements/elements.html');
+    if (!isInsideProjectFolder) {
+      this.emit('error', new Error(
+        'Element must be created inside a project folder'
```

Comment:
Sure. Man, I'm not comfortable writing in English, nor have I enough knowledge of javascript/polymer to correctly understand the code. Is there an option to suggest? or I have to commit and request a pull